site/content/docs/4.3/getting-started/webpack.md

```diff
@@ -26,7 +26,7 @@ import Alert from 'bootstrap/js/dist/alert';
 ...
 {{< /highlight >}}
 
-Bootstrap dependends on [Popper](https://popper.js.org/), which is speicified in the `peerDependencies` property.
+Bootstrap depended on [Popper](https://popper.js.org/), which is specified in the `peerDependencies` property.
```

Comment:
```suggestion
Bootstrap depends on [Popper](https://popper.js.org/), which is specified in the `peerDependencies` property.
```

Reply:
You have disabled edit from maintainers thus we can't make any further tweaks ourselves here.

Reply:
**edit from maintainers** is already enabled! I don't know what is stopping!

![Screenshot](https://user-images.githubusercontent.com/53072918/61576151-ac89d600-aaf3-11e9-9138-a19ceec62538.png)

Reply:
@XhmikosR looks to be a bug with GitHub, I've filed with them (#316968) but in the meantime edits through the branch UI still work.

@kittenking our apologies for the confusion and thank you for your contributions!